frontend/public/components/_terminal.scss

```diff
@@ -23,6 +23,7 @@ $console-collapse-link-z-index: $console-z-index + 20;
 
 .console-collapse-link {
   color: $color-pf-blue-300;
+  background:rgba(0, 0, 0,0.75) !important;
```

Comment:
nit: alphabetize and space after `:`

I guess `!important` is needed because `btn-link` sets background

Reply:
Fixed alphabetize and space